kinto_wizard/__main__.py

```diff
@@ -50,12 +52,27 @@ def main():
                            help="Export collections' records",
                            action='store_true')
 
+    # validate sub-command.
+    subparser = subparsers.add_parser('validate')
+    subparser.set_defaults(which='validate')
+    subparser.add_argument(dest='filepath', help='YAML file to validate')
+    cli_utils.add_parser_options(subparser)
+
     # Parse CLI args.
     args = parser.parse_args()
     cli_utils.setup_logger(logger, args)
     kinto_logger = logging.getLogger('kinto_http')
     cli_utils.setup_logger(kinto_logger, args)
 
+    if args.which == 'validate':
+        logger.debug("Start validation...")
+        logger.warning("Load YAML file {!r}".format(args.filepath))
+        with open(args.filepath, 'r') as f:
+            config = yaml.safe_load(f)
+            logger.warning("File loaded!")
```

Comment:
why warning? ...info, no?

Reply:
I used warning because I want it to be displayed without having to add a flag, maybe the default logger level should be INFO then, but currently I realized that if I leave it like that, I don't have any logs.

Reply:
I guess we found a bug